### ISSUE TITLE
adding a wx.media.MediaCtrl causes "Could not find a valid audiosink (error 0: Success)" to be displayed in a dialog

### ISSUE THREAD

**Issue by krichter722** [wxWidgets/Phoenix#245]
In the simplest case like

```
#!/usr/bin/python
# -*- coding: utf-8 -*-

import wx
import wx.media
import plac

class NoValidAudiosink(wx.Frame):

    def __init__(self, parent, id, title, input_file):
        wx.Frame.__init__(self, parent, id, title, size=(600, 500))
        self.videoPanel = wx.Panel(self)
        self.mplayerCtrl = wx.media.MediaCtrl(self.videoPanel, -1)
        self.Show(True)
        # the dialog is displayed already without the video being requested to be played
        self.mplayerCtrl.Load(input_file)
        self.mplayerCtrl.Play()

@plac.annotations(input_file=("the file to try to play", "positional"),
)
def __main_delegate__(input_file):
    """necessary function to make `plac.call` possible in `main`"""
    app = wx.App(False)
    frame = NoValidAudiosink(None, wx.ID_ANY, title="Title", input_file=input_file)
    frame.Show(True)
    app.MainLoop()

def main():
    """`entry_point` for `setuptools`"""
    plac.call(__main_delegate__)

if "__main__" == __name__:
    main()
```

A dialog with the message `Could not find a valid audiosink (error 0: Success)` is displayed right after start. I use the [Store Wars VLC demo file `sw_h264.asf`](http://altair.videolan.org/~dionoea/vlc-plugin-demo/streams/sw_h264.asf) from http://altair.videolan.org/~dionoea/vlc-plugin-demo/, i.e. invoke `python no_valid_audiosink.py /tmp/sw_h264.asf`. The user has no idea what the message could mean and how to proceed. I assume it's the reason that the `MediaCtrl` doesn't play. The message also occurs if the `MediaCtrl` is only displayed. The last part of the message `error 0: Success` is also post-factual.

experienced with a92cef7b on Ubuntu 16.10

**Comment by RobinD42** [comment]
Which platform?

**Comment by krichter722** [comment]
> Which platform?

Ubuntu 16.10

**Comment by krichter722** [comment]
This is fixed in 9b743cf3 and/or was caused by the `ext/wxWidgets` submodule pointing to a commit which wasn't requested by `master`.
